docs/internals/contributing/index.txt

```diff
@@ -3,53 +3,30 @@ Contributing to Django
 ======================
 
 Django is a community that lives on its volunteers. As it keeps growing, we
-always need more people to help others. As soon as you learn Django, you can
-contribute in many ways:
-
-* Join the `Django forum`_. This forum is a place for discussing the Django
-  framework and applications and projects that use it. This is also a good
-  place to ask and answer any questions related to installing, using, or
-  contributing to Django.
-
-* Join the |django-users| mailing list and answer questions. This
-  mailing list has a huge audience, and we really want to maintain a
-  friendly and helpful atmosphere. If you're new to the Django community,
-  you should read the `posting guidelines`_.
-
-* Join the `#django IRC channel`_ on Freenode and answer questions. By
-  explaining Django to other users, you're going to learn a lot about the
-  framework yourself.
-
-* Blog about Django. We syndicate all the Django blogs we know about on
-  the `community page`_; if you'd like to see your blog on that page you
-  can `register it here`_.
-
-* Contribute to open-source Django projects, write some documentation, or
-  release your own code as an open-source pluggable application. The
-  ecosystem of pluggable applications is a big strength of Django, help us
-  build it!
-
+always need more people to build Django. As a Django developer,
+you can contribute in many ways.
 If you think working *with* Django is fun, wait until you start working *on*
-it. We're passionate about helping Django users make the jump to contributing
-members of the community, so there are several ways you can help Django's
-development:
+it. so here are several ways you can help Django's development:
 
 * :doc:`Report bugs <bugs-and-features>` in our `ticket tracker`_.
 
 * Join the |django-developers| mailing list and share your ideas for how
   to improve Django. We're always open to suggestions. You can also interact
   on the `#django-dev IRC channel`_.
 
-* :doc:`Submit patches <writing-code/submitting-patches>` for new and/or
+* :doc:`Submit pull requests <submission/submitting-patches>` for new and/or
```

Comment:
As per main comment, let's defer worrying about precise locations until the new _Step this way…_ content is there, but I'm not sure about the `submission` path segment. Why can't it stay under `writing-code`? 🤔 (_…let's defer…_)

Reply:
Oh it's cause it's an information necessary to all contributors, developers, writers and translators. Putting a place where it is accessible by all.